SPECS/conda/conda.spec

```diff
@@ -0,0 +1,436 @@
+Requires:       python%{python3_pkgversion}-cytoolz >= 0.8.2
+Provides:       bundled(python%{python3_pkgversion}-appdirs) = 1.2.0
+Provides:       bundled(python%{python3_pkgversion}-auxlib)
+Provides:       bundled(python%{python3_pkgversion}-boltons) = 18.0.0
+Provides:       bundled(python%{python3_pkgversion}-six) = 1.10.0
+Provides:       bundled(python%{python3_pkgversion}-toolz) = 0.8.2
+
+%{?python_provide:%python_provide python%{python3_pkgversion}-conda}
+
+%description -n python%{python3_pkgversion}-conda %_description
+
+%prep
+%autosetup -p1
+
+sed -r -i 's/^(__version__ = ).*/\1"%{version}"/' conda/__init__.py
+
+# delete interpreter line, the user can always call the file
+# explicitly as python3 /usr/lib/python3.6/site-packages/conda/_vendor/appdirs.py
+# or so.
+sed -r -i '1 {/#![/]usr[/]bin[/]env/d}' conda/_vendor/appdirs.py
+
+# Use Fedora's cpuinfo since it supports more arches
+rm conda/_vendor/cpuinfo.py
+
+# Use system versions
+# TODO - urllib3 - results in test failures: https://github.com/conda/conda/issues/9512
+#rm -r conda/_vendor/{distro.py,frozendict.py,tqdm,urllib3}
+#find conda -name \*.py | xargs sed -i -e 's/^\( *\)from .*_vendor\.\(\(distro\|frozendict\|tqdm\|urllib3\).*\) import/\1from \2 import/'
+rm -r conda/_vendor/{distro.py,frozendict.py,tqdm}
+find conda -name \*.py | xargs sed -i -e 's/^\( *\)from .*_vendor\.\(\(distro\|frozendict\|tqdm\).*\) import/\1from \2 import/'
+
+%ifnarch x86_64
+# Tests on non-x86_64
+cp -a tests/data/conda_format_repo/{linux-64,%{python3_platform}}
+sed -i -e s/linux-64/%{python3_platform}/ tests/data/conda_format_repo/%{python3_platform}/*json
+%endif
+
+
+%build
+# build conda executable
+%define py_setup utils/setup-testing.py
+%py3_build
+
+%install
+# install conda executable
+%define py_setup utils/setup-testing.py
+%py3_install
+
+mkdir -p %{buildroot}%{_datadir}/conda/condarc.d
+cat >%{buildroot}%{_datadir}/conda/condarc.d/defaults.yaml <<EOF
+pkgs_dirs:
+ - /var/cache/conda/pkgs
+ - ~/.conda/pkgs
+EOF
+
+mkdir -p %{buildroot}%{_localstatedir}/cache/conda/pkgs/cache
+
+install -m 0644 -Dt %{buildroot}/etc/profile.d/ conda/shell/etc/profile.d/conda.{sh,csh}
+sed -r -i '1i CONDA_EXE=%{_bindir}/conda' %{buildroot}/etc/profile.d/conda.sh
+sed -r -i -e '1i set _CONDA_EXE=%{_bindir}/conda\nset _CONDA_ROOT=' \
+          -e 's/CONDA_PFX=.*/CONDA_PFX=/' %{buildroot}/etc/profile.d/conda.csh
+install -m 0644 -Dt %{buildroot}/etc/fish/conf.d/ conda/shell/etc/fish/conf.d/conda.fish
+sed -r -i -e '1i set -gx CONDA_EXE "/usr/bin/conda"\nset _CONDA_ROOT "/usr"\nset _CONDA_EXE "/usr/bin/conda"\nset -gx CONDA_PYTHON_EXE "/usr/bin/python3"' \
+          %{buildroot}/etc/fish/conf.d/conda.fish
+
+# Install bash completion script
+install -m 0644 -Dt %{buildroot}%{bash_completionsdir}/ %SOURCE1
+
+
+%check
+export PATH=%{buildroot}%{_bindir}:$PATH
+PYTHONPATH=%{buildroot}%{python3_sitelib} conda info
+
+# Integration tests generally require network, so skip them.
```

Comment:
We can try enabling these, because we do have network access during our test builds.

Reply:
No dice :(